tests/integration/blockchain/clients/midnight/client.rs

```diff
@@ -0,0 +1,136 @@
+use crate::integration::mocks::{
+	create_midnight_test_network_with_urls, create_midnight_valid_server_mock_network_response,
+	MockMidnightClientTrait, MockMidnightTransportClient,
+};
+use alloy::{
```

Comment:
Maybe these types should be rexported and used through the project if they suits other networks as well.

these are alloy evm types used for midnight.

in case of replacing library there would be a lot of changes.

If reexported as local ones you would need to change it at single place.

Reply:
Good catch. These are just placeholder and replaced in upcoming PRs, we only use alloy for EVM so far!